scipy/stats/_continuous_distns.py

```diff
@@ -1079,10 +1085,17 @@ class fisk_gen(burr_gen):
     -----
     The probability density function for `fisk` is:
 
+    .. math::
+    
+        f(x, c) = c x^{c-1} (1 + x^{c})^{-2}
+
+    Also equivalent to:
```

Comment:
This is actually confusing, without reading the linked issue it was non-obvious that this wasn't a mistake. I suggest the following rephrase: _Please note that the above expression can be transformed into the following one, which is also commonly used:_